core/database/build.gradle.kts

```diff
@@ -0,0 +1,12 @@
+plugins {
+    alias(libs.plugins.multimodulecrypto.android.library)
+    alias(libs.plugins.multimodulecrypto.android.room)
+    alias(libs.plugins.multimodulecrypto.android.hilt)
+}
+
+android {
+    namespace = "com.example.database"
```

Comment:
modülün namespace'i com.example.multimodulecrypto.core.database olmalıydı
Modüler yapıda her modülün adı app namespace'inin sonuna eklenmeli
Modül içerisindeki importlar bu isme göre isimlendirildiği için hiyerarşiyi importlarda da devam ettirmek önemli